packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/index.js

```diff
@@ -69,15 +71,62 @@ const confirmDialogsPropTypes = {
  * ConfirmDialogPublishAll
  * -----------------------------------------------------------------------------------------------*/
 
-const ConfirmDialogPublishAll = ({ isOpen, onToggleDialog, isConfirmButtonLoading, onConfirm }) => {
+const ConfirmDialogPublishAll = ({
+  isOpen,
+  onToggleDialog,
+  isConfirmButtonLoading,
+  onConfirm,
+  selectedEntries,
+  slug,
+}) => {
   const { formatMessage } = useIntl();
+  const { get } = useFetchClient();
+
+  const fetchDraftRelations = async () => {
+    try {
+      const endPoint = `/content-manager/collection-types/${slug}/actions/countManyEntriesDraftRelations?${stringify(
+        { ids: selectedEntries },
+        { encode: false }
+      )}`;
+
+      const draftRelations = await get(endPoint);
+      const draftEntries = await draftRelations.data.data;
+
+      return draftEntries;
+    } catch (err) {
+      console.error(err.message);
+
+      return null;
+    }
```

Comment:
This is not how react-query works. 
```suggestion
    try {
      const { data: { data } } = await get(`/content-manager/collection-types/${slug}/actions/countManyEntriesDraftRelations?${stringify(
        { ids: selectedEntries },
        { encode: false }
      )}`);
      
      return data
    } catch (err) {
      // silence
    }
```

Reply:
ok, thanks...done